sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -60,7 +60,7 @@ var client = new EventHubClient(connectionString, eventHubName);
 
 - A **consumer group** is a view of an entire Event Hub. Consumer groups enable multiple consuming applications to each have a separate view of the event stream, and to read the stream independently at their own pace and from their own position.  There can be at most 5 concurrent readers on a partition per consumer group; however it is recommended that there is only one active consumer for a given partition and consumer group pairing. Each active reader receives all of the events from its partition; ff there are multiple readers on the same partition, then they will receive duplicate events. 
 
```

Comment:
ff -> if

Reply:
Wow.... that one has been buried for a while.  Thanks!